docs/usage-cli.rst

```diff
@@ -174,3 +174,40 @@ You will see a similar result as in the screenshot below.
 .. figure:: img/usage_cli_imgcat.png
 
     Preview of DAG in iTerm2
+
+Formatting commands output
+--------------------------
+
+Some Airflow commands like ``dags list`` or ``tasks states-for-dag-run`` support ``--output`` flag which allow users
```

Comment:
```suggestion
Some Airflow commands like ``airflow dags list`` or ``airflow tasks states-for-dag-run`` support ``--output`` flag which allow users
```

This is really minor difference. But I'm trying to think from a 100% newbie perspective: "here is the command, I can simply copy and paste. Oops, why it doesn't work".

Reply:
Done in caf81ff

Reply:
Done in caf81ff